crates/bevy_input/src/keyboard.rs

```diff
@@ -326,8 +326,11 @@ pub enum KeyCode {
     LControl,
     /// The `LShift` / `Left Shift` key.
     LShift,
-    /// The `LWin` / `Left Windows` key. Maps to `Left Command` on Mac.
-    LWin,
+    /// The `LSuper` / `Left Super` key.
+    /// Generic keyboards usually display this key with the *Microsoft Windows* logo.
+    /// Apple keyboards call this key the *Command Key* and display it using the ⌘ character.
+    #[doc(alias("LWin", "LMeta", "LLogo"))]
+    LSuper,
```

Comment:
```suggestion
    SuperLeft,
```

We should align with #8745 to avoid conflicts (and keep consistency with upstream).

Reply:
Shouldn't we then also rename `LAlt`, `LBracket`, `LControl`, and `LShift` to `LeftAlt` etc.?

Reply:
`AltLeft`, but yes, I agree. I think that the increased clarity is worth the extra characters.

Reply:
I'm sure @mockersf has opinions though :p

Reply:
In that case we should also adjust the doc aliases to `WinLeft` etc.

Reply:
> I'm sure @mockersf has opinions though :p

If we change one, we should change them all... but no opinion on wether to do it now or with winit update later